### ISSUE TITLE
Provide Dockerfile / Docker image

### ISSUE THREAD

**Issue by PeterDaveHello** [smacke/subsync#68]
Not sure if you'd like to provide Dockerfile / Docker image so that subsync can be easily used without environment setup? I can help contribute Dockerfile if you want 🙂

**Comment by PeterDaveHello** [comment]
Oops, used a wrong label, looks like I don't have the permission to remove it.

**Comment by leonvsc** [comment]
I am looking for a dockerfile or docker image :) Maybe @PeterDaveHello can provide me his dockerfile if you have one?

**Comment by PeterDaveHello** [comment]
Yeah, I'll be happy to help, just want to confirm that if we can make it official.

**Comment by leonvsc** [comment]
Alright thanks. Do you have already a dockerfile? If yes, can you link it to me?

**Comment by smacke** [comment]
@PeterDaveHello no worries and I would love to provide a Dockerfile. Would you be willing to submit a pull request? Would be delighted to have your contribution.

**Comment by PeterDaveHello** [comment]
Sure thing I'd like to do, currently I'm tuning and testing the image, give me few days to prepare it.

**Comment by PeterDaveHello** [comment]
@leonvsc try this image please:

```
peterdavehello/subsync:testing
docker.pkg.github.com/peterdavehellokitchen/dockerregistry/subsync:testing
```

https://hub.docker.com/r/peterdavehello/subsync
https://github.com/PeterDaveHelloKitchen/DockerRegistry/packages/209353

Let me know if you got any issues

**Comment by leonvsc** [comment]
Thanks for the image. I will have a look into it. Can you provide me some documentation on how to use it? Especially how to run commands from subsync?

My ultimate goal was to integrate it via a [Docker mod](https://github.com/linuxserver/docker-mods) on the [linuxserver.io bazarr](https://hub.docker.com/r/linuxserver/bazarr) image. Bazarr has a postprocessing section where you can run commands when the subtitle is done downloading.

**Comment by PeterDaveHello** [comment]
> Thanks for the image. I will have a look into it. Can you provide me some documentation on how to use it? Especially how to run commands from subsync?

You could try this, replace `--version` with the parameters of subsync you want to use:

```sh
docker run --rm -it -v $PWD:/video --workdir /video peterdavehello/subsync:testing subsync --version
```

**Comment by leonvsc** [comment]
Thanks for the documentation.
I have tested it a little bit and i didnt ran into any issues. Maybe ask other people to test it also. I dont have a lot of free time atm.

**Comment by PeterDaveHello** [comment]
Please note that I've published the all images here:
https://hub.docker.com/repository/docker/peterdavehello/subsync

Available tags:
https://hub.docker.com/r/peterdavehello/subsync/tags

**Comment by Klench** [comment]
Will this Docker image contain an old version of FFsubsync as it was a while ago it was created? @PeterDaveHello can you provide with the Dockerfile you applied so one can create a new image that is up to date?

**Comment by PeterDaveHello** [comment]
I'd like to send a pull request for this project to add a Dockerfile. @smacke, does that look good to you?

**Comment by PeterDaveHello** [comment]
PR #226 sent, feel free to take a look!

**Comment by smacke** [comment]
addressed in https://github.com/smacke/ffsubsync/commit/c02ea78bc8b66b3450830efcd865b3cdd00d44b8 based off @PeterDaveHello's original PR, thanks for the contribution!
